modules/apps/frontend-css/frontend-css-cadmin-web/build.gradle

```diff
@@ -0,0 +1,25 @@
+buildCSS {
+	imports = [
+		new File(npmInstall.nodeModulesDir, "@clayui/css/src/scss")
+	]
+}
+
+dependencies {
+	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bnd.annotation", version: "4.2.0"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "default"
+	compileOnly group: "javax.portlet", name: "portlet-api", version: "3.0.1"
+	compileOnly group: "javax.servlet.jsp", name: "javax.servlet.jsp-api", version: "2.3.1"
+	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	compileOnly group: "org.osgi", name: "org.osgi.service.component", version: "1.3.0"
+	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
+	compileOnly project(":apps:portal-url-builder:portal-url-builder-api")
+	compileOnly project(":apps:static:osgi:osgi-util")
+	compileOnly project(":apps:static:portal-configuration:portal-configuration-metatype-api")
+	compileOnly project(":core:osgi-service-tracker-collections")
+	compileOnly project(":core:petra:petra-lang")
+	compileOnly project(":core:petra:petra-sql-dsl-api")
```

Comment:
Some imports are not being used... at least the dsl-api looks fishy. Can you check before sending?

It's just removing one line, compiling and if it doesn't compile, it's needed.

Checking the code I would say that...

```
compileOnly project(":apps:portal-url-builder:portal-url-builder-api")
compileOnly project(":apps:static:portal-configuration:portal-configuration-metatype-api")	
compileOnly project(":core:osgi-service-tracker-collections")	
compileOnly project(":core:petra:petra-lang")	
compileOnly project(":core:petra:petra-sql-dsl-api")
compileOnly project(":core:petra:petra-string")
```

are not being used and maybe some of the other ones too.

I can do the trimming if you like :)

Reply:
@nhpatt I'll resend with changes. Just let me know whatever I miss, this will be 100% trial and error 😆 .

Reply:
@nhpatt I removed all the gradle imports except for `compileOnly project(":apps:portal-url-builder:portal-url-builder-api")` . I think it's being used in https://github.com/liferay-frontend/liferay-portal/blob/88e3ff985870aeeaad6414a5be0f1e07c8c87ef3/modules/apps/frontend-css/frontend-css-cadmin-web/src/main/java/com/liferay/frontend/css/cadmin/web/internal/servlet/taglib/CadminTopHeadDynamicInclude.java#L21-L22.

Reply:
Nice :D